### PR TITLE
lightning: delete experimental warning in error resolution

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### What is changed, added or deleted? (Required)

Delete experimental warning in error resolution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
    1. https://github.com/pingcap/docs-cn/pull/8641 
    2. https://github.com/pingcap/docs-cn/pull/8642
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by glorv — APPROVED** [review]

File: tidb-lightning/tidb-lightning-error-resolution.md [hunk 1]
```diff
@@ -66,7 +62,21 @@ TiDB Lightning duplicate resolution can detect duplicate data only within the da
 
 ## Error report
 
-All errors are written to tables in the `lightning_task_info` database in the downstream TiDB cluster. After the import is completed, you can view the errors in the database and process them manually.
+If TiDB Lightning encounters an error during the import, it outputs a report in both the terminal and its log file when it exits.
```
Comment (glorv) [inline tidb-lightning/tidb-lightning-error-resolution.md:65]:
```suggestion
If TiDB Lightning encounters an error during the import, it outputs a summary in the terminal and a log at the end of its log file when it exits.
```

**Review by sunzhaoyang — APPROVED** [review]
LGTM

File: tidb-lightning/tidb-lightning-error-resolution.md [hunk 2]
```diff
@@ -66,7 +62,21 @@ TiDB Lightning duplicate resolution can detect duplicate data only within the da
 
 ## Error report
 
-All errors are written to tables in the `lightning_task_info` database in the downstream TiDB cluster. After the import is completed, you can view the errors in the database and process them manually.
+If TiDB Lightning encounters an error during the import, it outputs an error statistics summary in the terminal and a log at the end of its log file when it exits.
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-error-resolution.md:65]:
```suggestion
If TiDB Lightning encounters errors during the import, it outputs a statistics summary about these errors in both your terminal and the log file when it exits.
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

**Pull request merged**